controls/collectionview/grouping/delegate-group-descriptor.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Delegate Group Descriptor
+page_title: .NET MAUI CollectionView Documentation - Delegate Group Descriptor
+description: Review the Telerik UI for .NET MAUI CollectionView DelegateGroupDescriptor option which enables you to group by a custom key.
+position: 2
+slug: collectionview-delegate-group-descriptor
+tags: group, collectionview, groupdescriptor, dotnet maui, maui, grouping items
+---
+
+# .NET MAUI CollectionView Delegate Group Descriptor
+
+The `CollectionViewDelegateGroupDescriptor` enables you to group by a custom key (for example, some complex expression combining two or more properties) instead of being limited by the value of a single property. This descriptor exposes the following properties:
```

Comment:
```suggestion
The `CollectionViewDelegateGroupDescriptor` enables you to group the CollectionView items by a custom key (for example, some complex expression combining two or more properties) instead of by the value of a single property. This descriptor exposes the following properties:
```